module/VuFind/src/VuFind/Cover/Loader.php

```diff
@@ -302,6 +310,12 @@ protected function storeSanitizedSettings($settings)
     {
         $settings = array_merge($this->getDefaultSettings(), $settings);
         $this->isbn = new ISBN($settings['isbn'] ?? '');
+        $this->isbns = array_map(
```

Comment:
Should we array_filter this to ensure validity?

Reply:
isValid is called in getCleanISBNs in record driver, so there should not be any invalid isbn here

Reply:
Right!